app/assets/stylesheets/messages/base.css.scss

```diff
@@ -0,0 +1,17 @@
+.message {
+  margin-left: 3em;
+  margin-top: 1em;
+  text-align: left;
+
+  .message-header {
+
+  .button {
+    display: inline;
+    float: center;
+  }
+
+  .message-content {
```

Comment:
Line should be indented 4 spaces, but was indented 2 spaces<br>Selector should have depth of applicability no greater than 2, but was 3